Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -522,32 +531,104 @@ def alert_list_command(client: Client, args: dict) -> CommandResults | str:
+
+        demisto.debug('calling alert_workflow_update_request function')
+        response = client.alert_workflow_update_request(
+            alert_id, status, comment, determination, time_range, start, end, closure_reason)
+
+        demisto.debug(f'{response=}')
+
+        return PollResult(
+            partial_result=CommandResults(readable_output="running polling"),
+            response=None,
+            continue_to_poll=True,
+            args_for_next_run={"request_id": response['request_id']} | args
+        )
 
-    results = CommandResults(
-        outputs_prefix='CarbonBlackEEDR.Alert',
-        outputs_key_field='AlertID',
-        outputs=outputs,
-        readable_output=readable_output,
-        raw_response=result
-    )
-    return results
+    request_id = args['request_id']
+
+    demisto.debug('Calling the second endpoint')
+    response = client.alert_workflow_update_get_request(
+        request_id)
+    demisto.debug(f'{response=}')
+
+    request_status = response['status']
+    demisto.debug(f'{request_status=}')
+
+    if request_status == 'CREATED':
+        message = CommandResults(
+            readable_output="Checking again in 60 seconds...")
+        demisto.debug('returning PollResult with continue_to_poll=True')
+        return PollResult(
+            partial_result=message,
+            response=None,
+            continue_to_poll=True,
+            args_for_next_run={"request_id": request_id,
+
+                               **args})
+
+    elif request_status == 'COMPLETED':
+        changed_by = response['job_parameters']['job_parameters']['userWorkflowDto']['changed_by']
+        status_HR = response.get('job_parameters').get('job_parameters').get('request').get('status') if args.get('status') else None
+        message = CommandResults(
+            outputs={'AlertID': alert_id, 'ChangedBy': changed_by, 'Comment': args.get('comment'),
+                     'LastUpdateTime': response['last_update_time'], 'Status': status_HR},
+            outputs_prefix='CarbonBlackEEDR.Alert',
+            readable_output=tableToMarkdown(f'Successfully updated the alert: "{alert_id}"',
+                                            {'changed_by': changed_by,
+                                            'last_update_time': response['last_update_time'],
+                                            'determination': args.get('determination'),
+                                            'comment': args.get('comment'),
+                                            'closure reason': args.get('closure_reason'),
+                                            'status': status_HR}, removeNull=True))
+        demisto.debug('returning PollResult with continue_to_poll=False')
+        return PollResult(
+            response=message,
+            continue_to_poll=False)
+    
+    else:  # Status is failed
+        raise DemistoException('Failed to update the alerts workflow')
```

Comment:
```suggestion
        raise DemistoException("Failed to update the alerts workflow. Request's Status: {request_status}")
```

We dont have any other lgitimate status for polling? Running? Calculating? Processing?